include/gauge_field_order.h

```diff
@@ -1456,26 +1529,22 @@ namespace quda {
       for (int i=2; i<8; i++) out[i] = in[i];
     }
 
-    template<typename I>
+       template<typename I>
     __device__ __host__ inline void Unpack(RegType out[18], const RegType in[8], int idx, int dir,
                                            const RegType phase, const I *X, const int *R,
-                                           const Complex scale=Complex(static_cast<RegType>(1.0),
-                                                                       static_cast<RegType>(1.0))) const {
+                                           const Complex scale,
+                                           const Complex u) const {   
```

Comment:
trailing whitespace

Reply:
clang-format (I will just apply the latest working version to my changes and we can use that as a test)